Microsoft.Research/Contracts/System.Windows.Forms/System.Windows.Forms.DataGridViewColumn.cs

```diff
@@ -0,0 +1,338 @@
+        // </summary>
+        // 
+        // <returns>
+        // A <see cref="T:System.String"/> with the desired text. The default is an empty string ("").
+        // </returns>
+        // <PermissionSet><IPermission class="System.Security.Permissions.EnvironmentPermission, mscorlib, Version=2.0.3600.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" version="1" Unrestricted="true"/><IPermission class="System.Security.Permissions.FileIOPermission, mscorlib, Version=2.0.3600.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" version="1" Unrestricted="true"/><IPermission class="System.Security.Permissions.SecurityPermission, mscorlib, Version=2.0.3600.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" version="1" Flags="UnmanagedCode, ControlEvidence"/><IPermission class="System.Diagnostics.PerformanceCounterPermission, System, Version=2.0.3600.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" version="1" Unrestricted="true"/></PermissionSet>
+        // public string HeaderText {get; set;}
+
+        // <summary>
+        // Gets the sizing mode in effect for the column.
+        // </summary>
+        // 
+        // <returns>
+        // The <see cref="T:System.Windows.Forms.DataGridViewAutoSizeColumnMode"/> value in effect for the column.
+        // </returns>
+        // public DataGridViewAutoSizeColumnMode InheritedAutoSizeMode {get;}
+
+        // <summary>
+        // Gets the cell style currently applied to the column.
+        // </summary>
+        // 
+        // <returns>
+        // A <see cref="T:System.Windows.Forms.DataGridViewCellStyle"/> that represents the cell style used to display the column.
+        // </returns>
+        // [Browsable(false)]
+        // public override DataGridViewCellStyle InheritedStyle { get; }
+
+        /// <summary>
+        /// Gets a value indicating whether the column is bound to a data source.
+        /// </summary>
+        /// 
+        /// <returns>
+        /// true if the column is connected to a data source; otherwise, false.
+        /// </returns>
+        public bool IsDataBound {get;}
+        
+        /// <summary>
+        /// Gets or sets the minimum width, in pixels, of the column.
+        /// </summary>
+        /// 
+        /// <returns>
+        /// The number of pixels, from 2 to <see cref="F:System.Int32.MaxValue"/>, that specifies the minimum width of the column. The default is 5.
+        /// </returns>
+        /// <exception cref="T:System.ArgumentOutOfRangeException">The value is less than 2 or greater than <see cref="F:System.Int32.MaxValue"/>.</exception><PermissionSet><IPermission class="System.Security.Permissions.FileIOPermission, mscorlib, Version=2.0.3600.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" version="1" Unrestricted="true"/></PermissionSet>
+        [DefaultValue(5)]
+        public int MinimumWidth
+        {
+            get
+            {
+                Contract.Ensures(Contract.Result<int>() >= 2);
```

Comment:
Where would the constant live? If I place it in the class, it will introduce a crasanitizer error (as it doesn't exist in the class).